app/main.py

```diff
@@ -42,6 +54,21 @@ async def index(request: Request):
     return RedirectResponse(f"{BASE_PATH}/docs")
 
 
+@app.get(f"{BASE_PATH}/version")
+@limiter.limit(RATE_LIMIT)
+async def version(request: Request):
+    """
+    Return the current dataset version used by the API
+    The list of dataset releases is available at
+     https://github.com/ambanum/OpenTermsArchive-versions/releases
+    Also return the commit SHA on which the API was built
+    """
+    return {
+        "dataset_version": os.getenv("MOST_RECENT_DATASET", "unknown"),
+        "api_version": os.getenv("COMMIT_SHA", "unknown"),
+    }
```

Comment:
Can you please add lastUpdated

Reply:
This is what the `/version` route returns now:
<img width="1364" alt="Screenshot 2021-03-16 at 17 22 44" src="https://user-images.githubusercontent.com/30295971/111348695-b8d78500-8680-11eb-9217-a4569e3a2dda.png">